src/unstract/adapters/x2text/llama_parse/pyproject.toml

```diff
@@ -0,0 +1,26 @@
+[build-system]
+requires = ["pdm-backend"]
+build-backend = "pdm.backend"
+
+
+[project]
+name = "unstract-llama-parse-x2text"
+version = "0.0.1"
+description = "Llama Parse X2Text Adapter"
+authors = [
+    {name = "Zipstack Inc.", email = "devsupport@zipstack.com"},
+]
+dependencies = [
+    "llama-parse>=0.4.1",
```

Comment:
@harini-venkataraman please check with @gaya3-zipstack as well but I doubt if we are using these per adapter `pyproject.toml` files as intended. AFAIK, they are unused and can be removed / correctly used.
However the concern here is that all dependencies should go into the root `pyproject.toml`

Reply:
Yeah we are not using this one. Pls add any dependencies to the adapters->pyproject.toml